src/ert/cli/monitor.py

```diff
@@ -115,7 +114,7 @@ def _get_legends(self) -> str:
         latest_snapshot = self._snapshots[max(self._snapshots.keys())]
         total_count = len(latest_snapshot.reals)
         aggregate = latest_snapshot.aggregate_real_states()
-        for state_ in ALL_REALIZATION_STATES:
+        for state_ in REAL_STATE_TO_COLOR:
```

Comment:
This looks like it could change the order of states printed to the terminal?

Reply:
It did. So I changed the order in the dict to match the tests again.